tendrl-monitoring-integration.spec

```diff
@@ -116,6 +116,9 @@ py.test -v tendrl/monitoring_integration/tests || :
 
 
 %changelog
+* Thu Aug 23 2018 Shubhendu Tripathi <shtripat@redhat.com> - 1.6.3-11
```

Comment:
Release version in spec file(line 3) also needs to be updated

Reply:
My bad. Updating the PR soon